nav2_bt_navigator/include/nav2_bt_navigator/bt_navigator.hpp

```diff
@@ -30,34 +30,82 @@
 
 namespace nav2_bt_navigator
 {
-
+/**
+ * @class nav2_bt_navigator::BtNavigator
```

Comment:
I'd add a description about it being an action server

Reply:
How about this?
/**
 * @class nav2_bt_navigator::BtNavigator
 * @brief An action server that uses behavior tree for navigating a robot to its
 *  goal position.
 */

Reply:
LGTM